lmp/build.sh

```diff
@@ -82,16 +82,32 @@ rm -f ${DEPLOY_DIR_IMAGE}/*.txt
 rm -f ${DEPLOY_DIR_IMAGE}/*.wic
 
 # Link the license manifest for all the images produced by the build
-for img in ${DEPLOY_DIR_IMAGE}/*${MACHINE}.manifest; do
+for img in ${DEPLOY_DIR_IMAGE}/*${MACHINE}*.manifest; do
```

Comment:
Probably better to change to something like 'for img in `find ${DEPLOY_DIR_IMAGE} -name....`' instead, and handle the use case when manifest is not available outside this for.

Reply:
With find it's cleaner and the handle when manifest is not available can right after the finding. I will retest this in my factory

Reply:
done